css/about.css

```diff
@@ -0,0 +1,127 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
Nice job splitting the CSS files by page.  I like this approach and use it often so I don't have large CSS files I have to go through for multiple pages.